content/cpp/concepts/inclusion-guards/inclusion-guards.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'inclusion-guards'
+Description: 'Used in C++ to prevent multiple inclusions of the same header file.'
+Subjects:
+  - 'Computer Science'
```

Comment:
We can add a few more subjects from [subjects.md](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).

```suggestion
  - 'Code Foundations'
  - 'Computer Science'
```